lib/utils/config.js

```diff
@@ -1,14 +1,9 @@
-var trusted = [
-	"0x47e172f6cfb6c7d01c1574fa3e2be7cc73269D95",
-	"0xa42c9b0d1a30722aea8b81e72957134897e7a11a",
-	"0xa0af2e71cecc248f4a7fd606f203467b500dd53b",
-	"0x7f586ca3e80f71097fc28a2ecfef92ac47fb2263"
-]
+let trusted = []
 
-var banned = [];
+let banned = []
```

Comment:
ethstats is ES5 let shoudl error out on run. At least it did for me

Reply:
I'll use const